lib/e2e/src/project.rs

```diff
@@ -49,13 +51,32 @@ fn get_wasm(name: &str) -> eyre::Result<PathBuf> {
     let name = name.replace('-', "_");
     // Looks like
     // "rust-contracts-stylus/target/debug/deps/erc721-15764c2c9a33bee7".
-    let executable = env::current_exe()?;
-    let out_dir = executable
-        .parent()
-        .expect("executable path should have a final component")
-        .join("../../");
-    let wasm: PathBuf =
-        out_dir.join(format!("wasm32-unknown-unknown/release/{name}.wasm"));
+    let mut target_dir = env::current_exe()?;
+
+    // Recursively find a `target` directory.
```

Comment:
I would say this approach is much better for distributing library. But a bit hacky:)

Reply:
Yeah, we should probably find a better way in the future, I'm not that happy with it. Nonetheless, it should be fine for most use cases.